src/node/index.ts

```diff
@@ -180,13 +181,18 @@ export default function PluginInspect(options: Options = {}): Plugin {
       const id = args[0]
       const ssr = args[2]?.ssr
 
+      const start = Date.now()
       const _result = await fn.apply(context, args)
+      const end = Date.now()
 
       const result = typeof _result === 'object' ? _result?.id : _result
 
       const map = ssr ? idMapSSR : idMap
-      if (!id.startsWith('./') && result && result !== id)
-        map[id] = result
+      if (result && result !== id) {
```

Comment:
I removed the `id.startsWith('./')` since it would be nice to measure relative path resolve speed too. (Though I could also be persuaded not, Vite should handle this well)